src/Cassandra.IntegrationTests/TestClusterManagement/CcmCluster.cs

```diff
@@ -214,23 +214,17 @@ public void BootstrapNode(int nodeIdToStart, string dataCenterName, bool start =
         
         public void UpdateDseConfig(params string[] yamlChanges)
         {
-            if (yamlChanges == null) return;
-            var joinedChanges = string.Join(" ", yamlChanges.Select(s => $"\"{s}\""));
-            _ccm.ExecuteCcm($"updatedseconf {joinedChanges}");
+            _ccm.UpdateDseConfig(yamlChanges);
```

Comment:
This is a nice way to reuse existing code but it will result in a different behavior and possibly longer execution times because it will now execute a ccm command per line instead of a single ccm command for all of them.

Reply:
I think it's still better to reuse and delegate to `CcmBridge.UpdateConfig`. `CcmBridge.UpdateConfig` also fixes the yaml configs for C* 4.1+, too.
I can change `CcmBridge.UpdateConfig` to join the config strings and run only one ccm command.

Reply:
> I can change CcmBridge.UpdateConfig to join the config strings and run only one ccm command.

Yeah if this doesn't break stuff then it will be perfect 👍